server.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+
 require('dotenv').config();
 const express = require('express');
 const cors = require('cors');
```

Comment:
Hey there Kirk :)
This can be removed from frontend since your backend is managing these 👍